src/App/PropertyPythonObject.cpp

```diff
@@ -136,7 +136,9 @@ void PropertyPythonObject::fromString(const std::string& repr)
             state.apply(args);
         }
         else if (this->object.hasAttr("__dict__")) {
-            this->object.setAttr("__dict__", res);
+            if (res.isDict()) {
+                this->object.setAttr("__dict__", res);
+            }

```

Comment:
No need for an `else` clause here to handle the error condition?

Reply:
Changed it to `!res.isNone()`

Reply:
Thanks for the explanation, makes sense to me.